archinstall/lib/locale_helpers.py

```diff
@@ -26,12 +22,27 @@ def verify_keyboard_layout(layout):
 			return True
 	return False
 
+def verify_x11_keyboard_layout(layout):
+	for language in list_x11_keyboard_languages():
+		if layout.lower() == language.lower():
+			return True
+	return False
 
-def search_keyboard_layout(layout_filter):
+def search_keyboard_layout(layout):
```

Comment:
would prefer to not undo this change, since linter doesn't like shadowing filter.